openapi/accounts/paths/account-configuration.yaml

```
@@ -0,0 +1,66 @@
+patch:
+  summary: Configure Account
+  description: Set the client-configurable portions of an Account.
+  tags:
+    - Account
+  parameters:
+    - $ref: ../../shared/parameters.yaml#/AccountID
+    - $ref: ../../shared/parameters.yaml#/AcceptDatetimeFormat
+    - name: configureAccountBody
+      description: Representation of the Account configuration to set
+      in: body
+      schema:
+        type: object
+        properties:
+          alias:
+            type: string
+            description: Client-defined alias (name) for the Account
+          marginRate:
+            type: number
+            format: float
```

```suggestion
            format: decimal
```
These should be all decimals as the documentation suggests that these are returned as string but represent a decimal number